ur_description/config/ur5e/visual_parameters.yaml

```diff
@@ -1,67 +1,67 @@
 mesh_files:
   base:
     visual:
-      mesh: "package://ur_description/meshes/ur5e/visual/base.dae"
+      mesh: "ur5e/visual/base.dae"
       material:
         name: "LightGrey"
         color: "0.7 0.7 0.7 1.0"
     collision:
-      mesh: "package://ur_description/meshes/ur5e/collision/base.stl"
+      mesh: "ur5e/collision/base.stl"
 
   shoulder:
     visual:
-      mesh: "package://ur_description/meshes/ur5e/visual/shoulder.dae"
+      mesh: "ur5e/visual/shoulder.dae"
       material:
         name: "LightGrey"
         color: "0.7 0.7 0.7 1.0"
     collision:
-      mesh: "package://ur_description/meshes/ur5e/collision/shoulder.stl"
+      mesh: "ur5e/collision/shoulder.stl"
 
   upper_arm:
     visual:
-      mesh: "package://ur_description/meshes/ur5e/visual/upperarm.dae"
+      mesh: "ur5e/visual/upperarm.dae"
       material:
         name: "LightGrey"
         color: "0.7 0.7 0.7 1.0"
     collision:
-      mesh: "package://ur_description/meshes/ur5e/collision/upperarm.stl"
+      mesh: "ur5e/collision/upperarm.stl"
       mesh_files:
 
   forearm:
     visual:
-      mesh: "package://ur_description/meshes/ur5e/visual/forearm.dae"
+      mesh: "ur5e/visual/forearm.dae"
       material:
         name: "LightGrey"
         color: "0.7 0.7 0.7 1.0"
     collision:
-      mesh: "package://ur_description/meshes/ur5e/collision/forearm.stl"
+      mesh: "ur5e/collision/forearm.stl"
 
   wrist_1:
     visual:
-      mesh: "package://ur_description/meshes/ur5e/visual/wrist1.dae"
+      mesh: "ur5e/visual/wrist1.dae"
       material:
         name: "LightGrey"
         color: "0.7 0.7 0.7 1.0"
     collision:
-      mesh: "package://ur_description/meshes/ur5e/collision/wrist1.stl"
+      mesh: "ur5e/collision/wrist1.stl"
     visual_offset: -0.127
 
   wrist_2:
     visual:
-      mesh: "package://ur_description/meshes/ur5e/visual/wrist2.dae"
+      mesh: "ur5e/visual/wrist2.dae"
       material:
         name: "LightGrey"
         color: "0.7 0.7 0.7 1.0"
     collision:
-      mesh: "package://ur_description/meshes/ur5e/collision/wrist2.stl"
+      mesh: "ur5e/collision/wrist2.stl"
     visual_offset: -0.0997
```

Comment:
What are the offsets for? Can we add a comment why they are needed?

Reply:
This is the offset of the visual mesh to the coordinate system of the link. We could add a comment probably with the offiset axis here is set.